voilab-table.js

```diff
@@ -126,23 +145,29 @@ var lodash = require('lodash'),
         });
 
         self.pdf.y = pos.y + row._renderedContent.height;
-
-        self.pdf.moveDown(self.minRowHeight);
     },
 
     setRowHeight = function (self, row, isHeader) {
         var max_height = 0;
 
-        row._renderedContent = {data: {}, dataHeight: {}};
+        row._renderedContent = {data: {}, dataHeight: {}, contentHeight: {}};
 
         lodash.forEach(self.getColumns(), function (column) {
             var renderer = isHeader ? column.headerRenderer : column.renderer,
                 content = renderer ? renderer(self, row, false) : row[column.id],
-                height = !content ? 1 : self.pdf.heightOfString(content, lodash.assign(lodash.clone(column), {
-                    width: column.width - getPaddingValue('horizontal', column.padding)
-                })),
+                height = !content ? 1 : self.pdf.heightOfString(content, column),
```

Comment:
Oops, you're absolutely right. Can you confirm that it outputs the correct result when reverting this line back to the original?